### ISSUE TITLE
Exisiting Test spots the registration of a new test

### ISSUE THREAD

**Issue by StefanButscher** [corona-warn-app/cwa-app-android#4804]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name:  Fairphone 3
- Android version: 10  
- App version:  2.17.2

## Describe the bug
I do almost daily test and register the test using my Rapid-Test-Profil.
When I have an old test from the day before the registration gets stuck 
when I request the certificate.

When I then delete the old test, everything works fine. So this is rather
a matter of usability.

## Expected behaviour

Given I have an existing rapid test
When I register a new one, then the system asks me to delete the old one.

## Possible Fix

See above.

## Additional context

This is happening in Walldorf. If needed I can be there in person.

**Comment by mtwalli** [comment]
@StefanButscher  you can register tests in two ways in the app:
1- As family test, with this option you can register as many as you want without the need to delete old ones ,but you can't use any of positive tests under this option to warn others 
2- As personal test , with this option you can keep maximum one active test of PCR and Rapid antigen test. old tests under this option will be moved to recycle bin automatically when you register a new one of the same type (PCR , RAT). You can always restore them back from recycle bin ,but keep in mind that this will move active ones to the bin. 

TL;DR  User of the app can have maximum one active PCR and one active RAT , if user have already tests in the app they will moved to bin and stay there for 30 days afterwards they are deleted permanently. I hope it is clear now
